production/tanka/grafana-agent/v1/main.libsonnet.t

```diff
@@ -0,0 +1,142 @@
+local agent = import './internal/agent.libsonnet';
```

Comment:
Cleanup task: we can remove the production/tanka folder completely once we remove production/kubernetes